core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/PushLimitThroughSemiJoin.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
Updated the commit message
```
Avoid usage of Utils#transpose to pushdown LimitNode
because SemiJoinNode has 2 children whereas LimitNode
can have only 1 child.
```